app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for pers in people:
+        if pers["name"] not in Person.people:
+            Person(pers["name"], pers["age"])
```

Comment:
As this function is called only once, it's impossible there are some persons, so you can just create Person instances. Also you can use list comprehension here